deps/lua/src/lua_cjson.c

```diff
@@ -1131,7 +1131,7 @@ static void json_decode_descend(lua_State *l, json_parse_t *json, int slots)
 
     strbuf_free(json->tmp);
     luaL_error(l, "Found too many nested data structures (%d) at character %d",
-        json->current_depth, json->ptr - json->data);
+        json->current_depth, (int)(json->ptr - json->data));
```

Comment:
I would rather leave lua alone if we can.

Reply:
yeah, I know, but it's a vendored dependency and we would prefer it stay as it is upstream.